client/src/index.css

```diff
@@ -15,4 +15,34 @@
     background: linear-gradient(to right, #1626FF 29%, #FF0000 100%);
     background-clip: text;
     -webkit-text-fill-color: transparent;
-}
\ No newline at end of file
+}
+
```

Comment:
@abhishek-yeole 
Give the custom scrollbar class, as we don't want customize scrollbar for mobile screen 
![image](https://github.com/Saherpathan/GenUp_Nexus/assets/77800620/3458846b-2cca-4bab-8b3f-939a6e957b7d)
and use this class in the component or the page where you want to use custom scroll, like in CustomNode.jsx page

Reply:
and remove the class in the mobile screen

Reply:
Or do one thing just add @media for mobile device and remove styling in the scrollbar